packages/cc-ambient-types/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "../../schemas/module-config.schema.json",
+  "name": "@types/cc-ambient-types",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "build": "node ./scripts/build-consts.js"
+  },
+  "license": "MIT",
+  "types": "./src/index.d.ts",
+  "exports": {
+    ".": {
+      "node": "./src/index.d.ts",
+      "types": "./src/index.d.ts"
+    },
+    "./query": "./scripts/query.js"
```

Comment:
we add this `query` export to provide some interface to query info of the module, such `getDtsFiles()`